test/blackbox-tests/test-cases/pkg/helpers.sh

```diff
@@ -52,3 +52,17 @@ cat >dune.lock/lock.dune <<EOF
 (lang package 0.1)
 EOF
 }
+
+# [mkenvtestpkg <package> <depends> <env_update_op>] makes an opam <package> that depends
+# on <depends> and updated "env_var" using <env_update_op>
+mkenvtestpkg() {
```

Comment:
This looks like a function that isn't useful for all tests. So how about:

1.You move all of these related tests into a sub directory
2. Introduce a shell script with only this function.